src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -30,6 +31,46 @@ function InboxReason({inbox, fontSize = 'sm'}: Props) {
       ? `More than ${Math.round(count / EVENT_ROUND_LIMIT)}k`
       : `${count}`;
 
+  const getTooltipDescription = ({
+    until,
+    count,
+    window,
+    user_count,
+    user_window,
+  }: InboxReasonDetails) => {
+    if (until) {
+      // Was ignored until `until` has passed.
+      //`until` format: "2021-01-20T03:59:03+00:00"
+      return t('Was ignored until %(window)s', {
+        until: getFormattedDate(until, 'YYYY/MM/DD HH:mm:ss', {local: true}),
```

Comment:
I noticed that we usually use the format below:

```suggestion
        until: getFormattedDate(until, 'MMM D, YYYY LT', {local: true}),
```

Reply:
the `local` we should set according to the selected utc.

Reply:
I'm using `const utc = this.props.utc;` but didn't need to import or wrap anything using `withGlobalSelection`. Is it getting the data from a parent component? Surprised TS is not yelling at me about this.